### PR TITLE
Adding Javascript Piano

### LINKED ISSUE #616: Create a piano using HTML+CSS+JS
## **Describe the project**
We Use HTML For Creating A Structure For The Piano Keyboard And Styling Using CSS, Giving The Piano Effect On Buttons, And Did The Same Color Of The Piano Using Css. Lastly, We Use JavaScript For Piano Keyboard Sound Play Functionality.

## Issue Type:

- [ ] Bug Report
- [ ] Feature Request
- [ ] Documentation
- [ ] New Project

## **Additional context**
I want to do this project under SWOC23. Please assign me this issue.

## **Screenshots**

- 
Snapshot_22-01-28_06-06-09

### PR DESCRIPTION
**Pull request by Bishal011**
## 🛠️ Issue (#616)

Closes #616 

# Description

This project is made by using HTML, CSS and JAVASCRIPT.
We Use HTML For Creating A Structure For The Piano Keyboard.
Styling Using CSS, Giving The Piano Effect On Buttons, And Did The Same Color Of The Piano Using Css.
Lastly, We Use JavaScript For Piano Keyboard Sound Play Functionality.

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have made this from my own
- [x] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

- [x] My changes generate no new warnings
- [x] The title of my pull request is a short description of the requested changes.

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK

video -- 

https://user-images.githubusercontent.com/105370000/218316463-d14ac0d4-3abc-4068-b2a8-1b9386e51c52.mp4

Screenshots--

Screenshot (214)

### CODE REVIEW

**Comment by Bishal011** [comment]
A new user left a comment. This user must be approved by a Netlify team owner before comments can be displayed.

[Approve this user](https://app.netlify.com/teams/tusharkesarwani/reviewers)

**Review by zugzwang03 — APPROVED** [review]

**Review by Vanshiii203 — CHANGES_REQUESTED** [review]
Screenshot is not visible in the readme.

File: Projects/PIANO/Readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Styling Using CSS, Giving The Piano Effect On Buttons, And Did The Same Color Of The Piano Using Css.
+
+Lastly, We Use JavaScript For Piano Keyboard Sound Play Functionality.
+
+
+SCREENSHOT
+
+
+![](Screenshot%20(214).png)
```
Comment (Bishal011) [inline Projects/PIANO/Readme.md]: Screenshot (214)

**Comment by Bishal011** [comment]
Now please review this issue and merge the pull request.

**Review by Vanshiii203 — CHANGES_REQUESTED** [review]
Add the screenshot of project in readme

**Review by Vanshiii203 — APPROVED** [review]

**Comment by TusharKesarwani** [comment]
Copied project
https://codepen.io/gabrielcarol/pen/rGeEbY

**Pull request closed without merging**